lib/main.dart

```diff
@@ -8,8 +9,19 @@ void main() {
   runApp(MyApp());
 }
 
-class MyApp extends StatelessWidget {
+class MyApp extends StatefulWidget {
   // This widget is the root of your application.
+  @override
+  _MyAppState createState() => _MyAppState();
+}
+
+class _MyAppState extends State<MyApp> {
+  @override
+  void initState() {
+    super.initState();
+    CloudMessanger.cloudMessanger();
+  }
+
```

Comment:
Can we do this without changing `MyApp` to a stateful widget?
Since stateful widgets carry a performance overhead and there isn't really any state associated with `MyApp` that needs to be handled with a `StatefulWidget`.

Reply:
What if i use that directly in void main. I mean CloudMessanger directly in void main?

Reply:
That's exactly what I want you to do ;)